src/AnimateAsset.ts

```diff
@@ -0,0 +1,69 @@
+import * as Animate from './';
+import {Texture, Spritesheet} from 'pixi.js';
```

Comment:
`import type` for these?

Reply:
I tried that, but typedoc didn't understand, so I dropped it and am allowing Typescript to just recognize that it is a type only.

Reply:
That’s right. Type doc is still on 3.7.x